pkg/apis/cluster/v1alpha1/machineset_types.go

```diff
@@ -154,6 +177,17 @@ func (m *MachineSet) Validate() field.ErrorList {
 			errors = append(errors, field.Invalid(fldPath.Child("template", "metadata", "labels"), m.Spec.Template.Labels, "`selector` does not match template `labels`"))
 		}
 	}
+	validDelPolicy := false
```

Comment:
Is this validation actually needed? The openapi validation generated for the CRD already validates that the value is a string and one of the valid values.

Reply:
Good point. I will remove it.